Makefile

```diff
@@ -19,9 +25,9 @@ project:
 	@$(call check_for_gem,xcodeproj)
 	ruby Tests/Scripts/add_json_files.rb
 
-sourcery:
```

Comment:
Since sourcery is now also used to update `MongoSwift.swift` I renamed this to be more specific.